### DISCUSSION TITLE
GitHub code excaping issue / potential security issue?

### DISCUSSION THREAD

**Discussion by wendelin** [community/community#49536] [Discussions] [answered]
### Select Topic Area

Product Feedback

### Body

Some code is not correctly escaped and is breaking a discussion page's layout:

https://github.com/twbs/bootstrap/discussions/35695

![Screenshot 2023-03-08 at 18-38-34 An old question scroll overflow content in a flex-grow container · twbs_bootstrap · Discussion #35695](https://user-images.githubusercontent.com/4798094/223788474-78d473a4-82c2-4aac-a77d-f1f03e26bad9.png)

**Comment by LindseyB** [comment, marked as answer]
I'm not seeing this happening anymore. Is this still occuring for you?

**Reply by wendelin** [reply]
I guess they fixed it
